cb-talent-development/employer-style-base
# EM-1444: Rearrange Style Base

**Purpose:**
> As a developer I'd like to reorganize our Sass files so that it is faster and easier for any developer to find existing features and patterns in Employer and add new features to Employer that are consistent with our style base and make us look good

- Rearranges style base items and adds classes, but does not change any existing functionality
- Updates `README.md`
- Removes `sass/_lists.scss` and `sass/_reset.scss` which had been commented out of `_base.scss`

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-1444

**Screenshots**
n/a - there should be no changes

**QA Links:**
n/a

**How to Verify These Changes**

- Pull down this branch, [link your local copy of Employer](https://cb-content-enablement.atlassian.net/wiki/display/CE/Yarn), and verify that there are no errors. 
- If you're able to load an Employer page, everything is working!
- This PR should not change any implementations, variables, or mixins currently used in Employer

**Responsive considerations**
n/a

**Relevant PRs/Dependencies:**
n/a

**Additional Information**
